src/RpcServer/RpcServerSettings.cs

```diff
@@ -19,9 +20,32 @@ internal class Settings
         public string[] DisabledMethods { get; }
         public int MaxConcurrentConnections { get; }
 
-        public static Settings Default { get; private set; }
+        public RpcServerSettings(IPAddress bindAddress = null,
+            ushort port = 10332,
+            string sslCert = "",
+            string sslCertPassword = "",
+            string[] trustedAuthorities = null,
+            string rpcUser = "",
+            string rpcPass = "",
+            string masGasInvoke = "10",
```

Comment:
fees as `BigDecimal`?

Reply:
I was a little worried about that since the original ctor needs to know NativeContract.GAS.Decimals, but yeah you're right these should be BigDecimals. Will fix

Reply:
I changed maxGasInvoke and maxFee ctor params to be decimals.